.github/workflows/smoke-checks.yml

```diff
@@ -58,6 +58,8 @@ jobs:
     runs-on: macos-11
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 100
```

Comment:
Good one. My hopes here are we hopefully wont have PRs with branches containing that many commits :)

Reply:
🤞